aten/src/ATen/NestedTensorImpl.cpp

```diff
@@ -167,11 +169,11 @@ NestedTensorImpl::NestedTensorImpl(
     const caffe2::TypeMeta data_type,
     at::Tensor nested_size_tensor,
     at::Tensor nested_stride_tensor,
-    std::vector<int64_t>&& offsets)
+    at::Tensor offsets_tensor)
     : TensorImpl(std::move(storage), key_set, data_type),
       nested_size_tensor_(std::move(nested_size_tensor)),
       nested_stride_tensor_(std::move(nested_stride_tensor)),
-      storage_offsets_(std::move(offsets)),
+      offsets_tensor_(std::move(offsets_tensor)),
```

Comment:
Nit: I think now that everything is a tensor baking the type into the name is ugly ( I think this was partially my doing).  I like nested_size_, nested_stride_, storage_offsets_. I understand not wanting to do this and just observation

Reply:
I do agree with you; I'm going back and forth on whether I want to go through with it though :p

Reply:
Okay I did it